index.js

```diff
@@ -0,0 +1,56 @@
+window.document.querySelector('#note-form').addEventListener('submit', function(event) {
+    event.preventDefault(); 
+
+    const title = window.document.querySelector("#note-title").value;
+    const content = window.document.querySelector("#note-content").value;
+    const priority = window.document.querySelector('#note-priority').value;
+
+    const note = document.createElement('div');
+    note.classList.add('note');
+
+    const noteTitle = document.createElement('div');
+    noteTitle.classList.add('note-title');
+    noteTitle.innerText = title;
+
+    const noteContent = document.createElement('div');
+    noteContent.classList.add('note-content');
+    noteContent.innerText = content;
+
+    const notePriority = document.createElement('div');
+    notePriority.classList.add('note-priority');
+    notePriority.innerText = `Priority: ${priority.charAt(0).toUpperCase() + priority.slice(1)}`;
```

Comment:
Missing no priority value validation

- [ ] Fix priority value to render `No priority` if not exists.